code/modules/goonchat/browserOutput.dm

```diff
@@ -233,6 +233,8 @@ GLOBAL_DATUM_INIT(iconCache, /savefile, new("tmp/iconCache.sav")) //Cache of ico
 			if (!C)
 				continue
 
+			C.mob.log_message(message, LOG_TOCHAT)
```

Comment:
add an argument to the to_chat proc, to choose whether to log the message or not, defaulting to not, please.

Logging to_chat should be opt-in on a message to message basis.

Reply:
I just had a think;

Consider making the extra argument be the "name" of the log, as in:

```
/proc/to_chat(target, message, handle_whitespace=TRUE, logname)

    [...]

   if(logname)
         C.mob.log_message(logname + ": " + message, LOG_TOCHAT)
```

That way we can add `flavourtext` as an argument to to_chat, and we can then search for `flavourtext` in the log if we need to access it